hammer/README.md

```diff
@@ -44,3 +44,44 @@ go run ./hammer \
   --leaf_write_goal=2500 \
   --show_ui=false
 ```
+
+# Design
```

Comment:
nit: in some places I've seen `#` be described as a "title level" heading with guidance that there should only be one of these in a given doc - if you subscribe to that philosophy maybe these headings should all be +1 `#`?

Reply:
I use markdown a lot and used to subscribe to this philosophy, but then I got bored of excessive indenting and running out of distinguishable indent levels once you get past 3. I can indent everything more if that's the team decision though?